cwms_radar_api/src/main/java/cwms/radar/api/CatalogController.java

```diff
@@ -123,18 +138,23 @@ public void getOne(Context ctx, String dataSet) {
             String unitSystem = ctx.queryParamAsClass("unitSystem",String.class).getOrDefault(UnitSystem.SI.getValue());
             Optional<String> office = Optional.ofNullable(
                                          ctx.queryParamAsClass("office", String.class).allowNullable()
-                                            .check( ofc -> Office.validOfficeCanNull(ofc), "Invalid office provided" )
+                                            .check(Office::validOfficeCanNull, "Invalid office provided" )
```

Comment:
This syntax is still visually odd, but it's definitely growing on me.

Reply:
Before I submit sonarlint warns me about issues.  I don't really care strongly one way or the other wrt the syntax but after I've looked at the same warning 4-5 times in a day I just give in and change it just to make the warning disappear.